scriptmodules/supplementary/golang.sh

```diff
@@ -0,0 +1,45 @@
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+rp_module_id="golang"
+rp_module_desc="Golang binary install"
+rp_module_section="opt"
+
+function depends_golang() {
+    getDepends curl tar
```

Comment:
Tar is not needed as a dependency (system depends on it).

Please use wget rather than curl for consistency with other modules (it is also not needed as a dependency).